packages/beacon-node/test/unit/chain/rewards/blockRewards.test.ts

```diff
@@ -89,7 +89,10 @@ describe("chain / rewards / blockRewards", () => {
       // Populate tree root caches of the state
       state.hashTreeRoot();
       cachedStateAltairPopulateCaches(state);
-      const calculatedBlockReward = await computeBlockRewards(block.message, state as CachedBeaconStateAllForks);
+      const calculatedBlockReward = await computeBlockRewards(
+        block.message,
+        state as unknown as CachedBeaconStateAllForks
```

Comment:
I don't think we need to cast state as `unknown` anymore. This is solved by #6646